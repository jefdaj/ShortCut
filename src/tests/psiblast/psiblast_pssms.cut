# Train a pssm per gene against a list of genomes, search them all against one
# genome, and return a list of hit tables. Depending on your specific purpose
# you might want priblast_pssms_all, which returns one big table.

queries    = split_faa (load_faa "data/7942genes10.faa")
pcc6803    = load_faa "data/6803genes.faa"
pcc7942    = load_faa "data/7942genes.faa"
cyanodb    = makeblastdb_prot [pcc6803, pcc7942]
pssms      = psiblast_train_pssms_db 1e-10 queries cyanodb

# TODO write/rewrite:
tables6803 = psiblast_pssms          1e-10 pssms pcc6803

result     = tables6803
