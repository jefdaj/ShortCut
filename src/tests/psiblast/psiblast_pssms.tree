.
|-- cache
|   |-- lines
|   |   |-- 07e55b9c0c.txt
|   |   |-- 0940002186.txt
|   |   |-- 0cf27272d3.txt
|   |   |-- 387ffff414.txt
|   |   |-- 3f62a639a8.txt
|   |   |-- 3fc39fbacc.txt
|   |   |-- 4d74902f43.txt
|   |   |-- 5d76f214eb.txt
|   |   |-- 7af3848d3d.txt
|   |   |-- 7c3bd40fa0.txt
|   |   |-- 92b0f8b713.txt
|   |   |-- 95c56cba65.txt
|   |   |-- a1512fc008.txt
|   |   |-- ab328cf65d.txt
|   |   |-- adfdb6bd8c.txt
|   |   |-- b67351f934.txt
|   |   |-- bdc783547b.txt
|   |   |-- c3ae472116.txt
|   |   |-- ef87d3e725.txt
|   |   |-- f0af818a51.txt
|   |   `-- f189fa94c1.txt
|   |-- load
|   |   |-- 32841598b2.faa -> $WORKDIR/data/7942genes10.faa
|   |   |-- 8aa4a9d2fc.faa -> $WORKDIR/data/6803genes.faa
|   |   `-- a810086108.faa -> $WORKDIR/data/7942genes.faa
|   |-- makeblastdb
|   |   |-- a1512fc008
|   |   |   |-- a1512fc008.pdb.list.phr
|   |   |   |-- a1512fc008.pdb.list.pin
|   |   |   `-- a1512fc008.pdb.list.psq
|   |   `-- c3ae472116
|   |       |-- c3ae472116.pdb.phr
|   |       |-- c3ae472116.pdb.pin
|   |       `-- c3ae472116.pdb.psq
|   |-- map
|   |   |-- 04855525fe.pssm -> ../../cache/lines/7c3bd40fa0.txt
|   |   |-- 05ac75b513.pssm -> ../../cache/lines/95c56cba65.txt
|   |   |-- 09aad1c905.bht
|   |   |-- 0d240369ea.bht
|   |   |-- 1926159518.pssm -> ../../cache/lines/387ffff414.txt
|   |   |-- 1e3966ffd2.pssm -> ../../cache/lines/f189fa94c1.txt
|   |   |-- 2e680897b5.pssm -> ../../cache/lines/b67351f934.txt
|   |   |-- 4521b681e0.bht
|   |   |-- 587c857b88.pssm -> ../../cache/lines/5d76f214eb.txt
|   |   |-- 5d68224585.pssm -> ../../cache/lines/07e55b9c0c.txt
|   |   |-- 5e0f59d21e.bht
|   |   |-- 5fb4963d4b.bht
|   |   |-- 67c2715c4f.bht
|   |   |-- 6eaca68213.bht
|   |   |-- a08a71c423.bht
|   |   |-- ac4b623aa9.pssm -> ../../cache/lines/ab328cf65d.txt
|   |   |-- b0338f5c0b.bht
|   |   |-- e5c5edbabd.pssm -> ../../cache/lines/0940002186.txt
|   |   |-- ebfbc86ad4.bht
|   |   `-- fdd49117fc.pssm -> ../../cache/lines/3fc39fbacc.txt
|   `-- split_faa
|-- exprs
|   |-- concat_bht
|   |   `-- 8f7061b4ac.bht
|   |-- list
|   |   |-- 7d562183ce.faa.list -> ../../cache/lines/c3ae472116.txt
|   |   `-- 94f9daed94.faa.list -> ../../cache/lines/a1512fc008.txt
|   |-- load_faa
|   |   |-- 18a54fafca4.faa
|   |   |-- 2d88028e40.faa -> ../../cache/load/32841598b2.faa
|   |   |-- 387f499ddc.faa -> ../../cache/load/a810086108.faa
|   |   |-- 55895cbd9ee.faa
|   |   |-- 5b316091639.faa
|   |   |-- 6f29186ec90.faa
|   |   |-- 82645ed6f61.faa
|   |   |-- 9500e84b092.faa
|   |   |-- a3a42267174.faa
|   |   |-- b2a261c0c1f.faa
|   |   |-- cab744a6f9d.faa
|   |   |-- dee24d6c3c.faa -> ../../cache/load/8aa4a9d2fc.faa
|   |   `-- e225e53d82b.faa
|   |-- makeblastdb_prot_all
|   |   |-- 27a934c58e.pdb.list -> ../../cache/lines/7af3848d3d.txt
|   |   `-- a76d25f4b5.pdb -> ../../cache/lines/3f62a639a8.txt
|   |-- num
|   |   `-- 3e281fad0f.num -> ../../cache/lines/0cf27272d3.txt
|   |-- psiblast_each_pssm
|   |   `-- 5fbeb4e044_ef7788b175_28e0df471c.bht.list -> ../../cache/lines/4d74902f43.txt
|   |-- psiblast_train_pssms_db
|   |   `-- 5fbeb4e044_a726be61de_e15f764d92.pssm.list -> ../../cache/lines/92b0f8b713.txt
|   |-- split_faa
|   |   `-- 1af4d085e7.faa.list -> ../../cache/lines/f0af818a51.txt
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/bdc783547b.txt
|       |-- db90f32127.str -> ../../cache/lines/ef87d3e725.txt
|       `-- ec3b181d8a.str -> ../../cache/lines/adfdb6bd8c.txt
`-- vars
    |-- cyanodb.pdb -> ../exprs/makeblastdb_prot_all/a76d25f4b5.pdb
    |-- pcc6803.faa -> ../exprs/load_faa/dee24d6c3c.faa
    |-- pcc7942.faa -> ../exprs/load_faa/387f499ddc.faa
    |-- pssms.pssm.list -> ../exprs/psiblast_train_pssms_db/5fbeb4e044_a726be61de_e15f764d92.pssm.list
    |-- queries.faa.list -> ../exprs/split_faa/1af4d085e7.faa.list
    |-- result -> ../vars/tables6803.bht
    `-- tables6803.bht -> ../exprs/concat_bht/8f7061b4ac.bht

19 directories, 82 files
