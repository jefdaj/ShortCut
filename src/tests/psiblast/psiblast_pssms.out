Synpcc7942_0001	BAA16871.1	57.179	390	149	5	208	580	1	389	7.07e-149	433
Synpcc7942_0003	BAA16646.1	74.809	786	178	9	187	966	4	775	0.0	1146
Synpcc7942_0004	BAA10132.1	67.992	503	151	4	185	685	1	495	0.0	681
Synpcc7942_0004	BAA16727.1	26.033	242	159	7	219	451	2	232	6.11e-12	65.5
Synpcc7942_0005	BAA10081.1	64.896	527	178	1	199	718	8	534	0.0	684
...

